evap/staff/forms.py

```diff
@@ -178,7 +179,7 @@ def clean(self):
         vote_start_date = self.cleaned_data.get('vote_start_date')
         vote_end_date = self.cleaned_data.get('vote_end_date')
         if vote_start_date and vote_end_date:
-            if vote_start_date >= vote_end_date:
+            if vote_start_date.date() >= vote_end_date:
```

Comment:
hah no, i just saw you changed this in contributor/forms.py. this here is the other place where this needs to be applied :)